modules/ROOT/pages/_partials/sync/openshift-example-app.adoc

```diff
@@ -1,3 +1,25 @@
 = Running Example Application
 
-TODO
\ No newline at end of file
+To quickly test {sync-service} service our sample docker image can be used. 
+
+.Fill in this as a parameters for provisioning
+[options="header"]
+|====
+|Field|Value
+|Sync Application Name|mysyncapp
+|Sync Application Docker image|http://docker.io/aerogear/voyager-server-example-task
+|Sync App Port|4000
+|SQL Server Hostname|db.example.com
+|SQL Server Port|1234
+|SQL Server Username|test
+|Database name|test
+|====
+Note, this image doesn't use any database, so values doesn't matter there. For other values, don't fill anything, default will be used.
```

Comment:
@wojta I had the impression it needs a DB. For example: https://github.com/aerogear/apollo-voyager-ionic-example/blob/master/server/src/schema.js#L36

Reply:
@wojta what's the basic example then? can you point me at the repo?

Reply:
It's just this one https://github.com/aerogear/apollo-voyager-server/tree/master/examples/basic.

Reply:
It now points to aerogear/voyager-server-example-basic.